src/components/InstancesList/InstancesList.js

```diff
@@ -799,7 +799,8 @@ class InstancesList extends React.Component {
 
     const itemToView = getItem(`${namespace}.position`);
 
-    const missedMatchItem = (query) => {
+    const missedMatchItem = (item) => {
+      const query = item && new URLSearchParams(this.props.location.search).get('query');
```

Comment:
This will override values passed from `subject` and `contributor`
I think this should use `||` instead of `&&` - if `item` is passed then it'll be rendered, otherwise get value from url

Reply:
Also, this returns an issue described here, but for call number browse
https://issues.folio.org/browse/UIIN-2087

Reply:
I reverted this part, looks like it was unneeded work